packages/template-retail-react-app/app/components/product-tile/promo-callout.jsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {findLowestPrice} from '@salesforce/retail-react-app/app/utils/product-utils'
+
+const PromoCallout = ({product}) => {
+    const {minPrice, data} = findLowestPrice(product)
+
+    // NOTE: API inconsistency - with getProduct call, a variant does not have productPromotions
+    const promos = data.productPromotions ?? product.productPromotions ?? []
+    const promo = promos.find((promo) => promo.promotionalPrice === minPrice) ?? promos[0]
```

Comment:
This seem to be repetitive code/execution. Can we return the applicable promotion as well in the` findLowestPrice(product)` method?